lib/msal-core/src/ScopeSet.ts

```diff
@@ -160,4 +153,71 @@ export class ScopeSet {
 
     // #endregion
 
+    /**
+     * @ignore
+     * Returns true if the scopes array only contains openid and/or profile
+     */
+    static onlyContainsOidcScopes(scopes: Array<string>): boolean {
+        let oidcScopesFound = 0;
+
+        if (scopes.indexOf(Constants.openidScope) > -1) {
+            oidcScopesFound += 1;
+        }
+
+        if (scopes.indexOf(Constants.profileScope) > -1) {
+            oidcScopesFound += 1;
+        }
+
+        const nonOidcScopes = scopes.length - oidcScopesFound;
+
+        return (nonOidcScopes === 0);
+
+    }
+
+       /**
+     * @ignore
+     * Returns true if the scopes array only contains openid and/or profile
+     */
+    static containsAnyOidcScopes(scopes: Array<string>): boolean {
+        const containsOpenIdScope = scopes.indexOf(Constants.openidScope) > -1;
+        const containsProfileScope = scopes.indexOf(Constants.profileScope) > -1;
+
+        return (containsOpenIdScope || containsProfileScope);
+    }
+
+    /**
+     * @ignore
+     * Returns true if the clientId is the only scope in the array
+     */
+    static onlyContainsClientId(scopes: Array<String>, clientId: string): boolean {
+        return (scopes) ? (scopes.indexOf(clientId) > -1 && scopes.length === 1) : false;
```

Comment:
Doesn't need to be a ternary
```suggestion
        return scopes && (scopes.indexOf(clientId) > -1) && (scopes.length === 1) ;
```

Reply:
I thought so too, but I did some testing and it turns out when scopes is null this would actually return null and not false (unless you double negate it).

Reply:
Can you do `!!scopes` ?

Reply:
Oh didn't see your note about double negation

Reply:
Do you prefer `!!scopes`? I have no preference, personally.

Reply:
I don't feel strongly about it but I think I slightly prefer just doing a straight boolean check over a ternary. I'll leave the ultimate decision up to you or if someone else feels more strongly one way or the other.

Reply:
I made the change, adding a comment about the double negation to make it clearer!